README.md

```diff
@@ -424,6 +429,26 @@ This option is only useful if the debug output secret `ACTIONS_STEP_DEBUG` is se
 
 Default value: `true`
 
+#### ignore-all-activities-before-stale
+
+The option [days-before-stale](#days-before-stale) will define the number of days before considering the issues or the pull requests as stale.  
+In most cases, the purpose of this action is to only stale when necessary so if any update occurs or if a comment is added to them, the counter will restart.  
+Nonetheless, if you don't care about this, and you prefer to stick to this number of days no matter the activity, you can enable this option.  
+Instead of comparing the number of days base on the [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`, it will be based on the [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `created_at`.
```

Comment:
```suggestion
Instead of comparing the number of days based on the [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`, it will be based on the [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `created_at`.
```